### DISCUSSION TITLE
Stop signs possible?

### DISCUSSION THREAD

**Discussion by tethlah** [slyh/Cities2-TrafficLightsEnhancement#5] [Ideas]
Is it possible to add 2 way stop signs to this?  I don't know anything about the structure of how the game is coded, are stop signs their own thing or are they part of a "traffic control" class or something where all way stop signs can be changed to 2 way?

**Comment by slyh** [comment]
I have a feeling that CO will implement this feature eventually because it's present in CS1 iirc. Therefore, it will be at the very end of the backlog. I will revisit this once the major features are completed.
